Framework/PythonInterface/mantid/plots/surfacecontourplots.py

```diff
@@ -40,11 +40,9 @@ def plot(plot_type: SpectraSelection, plot_index: int, axis_name: str, log_name:
 
             fig.show()
         elif plot_type == SpectraSelection.Contour:
-            fig = pcolormesh([matrix_ws])
+            fig = pcolormesh([matrix_ws], contour=True)
```

Comment:
Can you help me understand why we are merging `ax.pcolormesh` with `ax.contour` like this? A contour has different options and this starts to blur these lines a bit.

Reply:
I just thought that you would only ever have contour lines over a colorfill plot as that was the case with the advanced plots menu, but I can separate them out to make things cleaner.